refextract/references/pdf.py

```diff
@@ -79,7 +79,7 @@ def sortfunc(dest_couple):
             # extract the TeXkey from the named destination name
             return [re_reference_in_dest.match(destname).group(1)
                     for (destname, _) in refs]
-        except IncompleteCoordinatesError:
+        except Exception:
             print("* PDF: Impossible to determine layout, no TeXkeys returned")
```

Comment:
Maibe it should do like https://github.com/inspirehep/refextract/pull/49/files#diff-0ca191a5463455f7935dcccb48a79be8R56 ? so we don't lose the exception.

Reply:
Sure, done. Although https://github.com/inspirehep/refextract/issues/19 is really needed if we ever want to make use of this exception...

Reply:
That would be best yep, so we get it in sentry.